README.md

```diff
@@ -600,6 +600,24 @@ when creating the workload otherwise the workload will always finish with `Compl
     `125`: Workload finished but did not complete successfully.
     `1`: Other failure.
 
+## Job List
+
+*   Job List (see Slurm jobs):
```

Comment:
Where I can see? 🤔

Reply:
Maybe I misunderstood but the text in () is just a very short description of what Job List does.

Reply:
Sure. Should I remove it as well from the command description?

Reply:
I think so.

Reply:
Done.

Reply:
Thanks!